portal/client/src/scss/vendor.scss

```diff
@@ -0,0 +1,10 @@
+@import './variables';
+@import "~@fortawesome/fontawesome-free/css/all.css";
+@import "~@fortawesome/fontawesome-free/css/fontawesome.css";
+
+@import '~bulma/bulma.sass';
+@import "~bulma-extensions/bulma-checkradio/src/sass/index.sass";
+@import '~bulma-extensions/bulma-calendar/src/sass/index.sass';
+@import '~bulma-extensions/bulma-tooltip/src/sass/index.sass';
```

Comment:
are you planning to use these components?

Reply:
nope